akka-bbb-apps/src/main/scala/org/bigbluebutton/core/db/PresPresentationDAO.scala

```diff
@@ -99,4 +103,15 @@ object PresPresentationDAO {
       }
   }
 
+  def updateErrorMsgKey(presentationId: String, errorMsgKey: String) = {
+    DatabaseConnection.db.run(
+      sqlu"""UPDATE pres_presentation SET
+            "errorMsgKey" = $errorMsgKey
+            WHERE "presentationId" = $presentationId"""
```

Comment:
For this case, I suggest using the Slick update function to do the update.
For example: https://github.com/bigbluebutton/bigbluebutton/blob/5e2f6ac777f72008ff119670774f0ff1f09d5538/akka-bbb-apps/src/main/scala/org/bigbluebutton/core/db/UserDAO.scala#L82